django/db/models/query.py

```diff
@@ -136,6 +137,29 @@ def __iter__(self):
         return compiler.results_iter(tuple_expected=True)
 
 
+class NamedValuesListIterable(ValuesListIterable):
+    """
+    Iterable returned by QuerySet.values_list(named=True) that yields a
+    namedtuple for each row.
+    """
+
+    @staticmethod
+    @lru_cache()
+    def create_namedtuple_class(*names):
+        return namedtuple('Row', names)
```

Comment:
Did you consider using `queryset.model` instead of 'Row'? I don't know if that would cause confusion with model instances.

Reply:
IMHO it doesn't make much sense as returned tuples don't have strong connection to model. User could do something like this:
```
Model.objects.annotate(f=models.F('related__f')).values_list('f', named=True)
```